index.js

```diff
@@ -150,6 +170,8 @@ class OError extends Error {
  */
 class TaggedError extends OError {}
 
+const DROPPED_TAGS_ERROR = new TaggedError('... dropped tags')
```

Comment:
Is it worth documenting this as `@private`? (And possibly move it just under `OError.maxTags`?)

Reply:
Since it's not exported and doesn't seem to be picked up by typescript (or jsdoc), I think it is OK. (I had to add jsdoc comments to some other non-exported members, e.g. `indent`, to satisfy typescript in `--strict` mode, and I put `@private` on those to make it clear, but I don't think we need to tag everything that is not exported as private.)

I tried moving it up, but it has to come after `TaggedError` is defined. I could move `OError.maxTags` down, but then it would be further from the class body. (And for the record we did try making it a static property in the class body, but it seems that is still an experimental language feature.)